mmv1/products/vpcaccess/api.yaml

```diff
@@ -105,3 +127,26 @@ objects:
         description: |
           The fully qualified name of this VPC connector
         output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'subnet'
+        input: true
+        min_version: beta
+        description: |
+          The subnet in which to house the connector
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'name'
+            description: |
+              Subnet name (relative, not fully qualified). E.g. if the full subnet selfLink is
+              https://compute.googleapis.com/compute/v1/projects/{project}/regions/{region}/subnetworks/{subnetName} the correct input for this field would be {subnetName}"
+            exactly_one_of:
+              - network
+              - subnet.0.name
+            input: true
+            min_version: beta
```

Comment:
```suggestion
          - !ruby/object:Api::Type::String
            name: 'name'
            description: |
              Subnet name (relative, not fully qualified). E.g. if the full subnet selfLink is
              https://compute.googleapis.com/compute/v1/projects/{project}/regions/{region}/subnetworks/{subnetName} the correct input for this field would be {subnetName}"
            exactly_one_of:
              - network
              - subnet.0.name
            input: true
            min_version: beta
            required: true
```

Reply:
adding required to subnet.0.name so user can't have an empty subnet block in their config.

Reply:
I should probably open an issue for mmv1 to validate that required and exactly_one_of isn't set on the same field.

Reply:
It doesn't work.

```
    provider_test.go:270: Step 1/1 error: Error running pre-apply refresh: 
        Error: InternalValidate
        
        Internal validation of the provider failed! This is always a bug
        with the provider itself, and not a user issue. Please report
        this bug:
        
        1 error occurred:
        	* resource google_vpc_access_connector: ExactlyOneOf: network cannot contain
        Required attribute (subnet.0.name)
```

Reply:
:o thanks for checking, I'm definitely in the wrong here! Thanks for you help